src/api/v1/community_businesses/visit_activities.ts

```diff
@@ -0,0 +1,130 @@
+import * as Hapi from 'hapi';
+import * as moment from 'moment' ;
+import { CommunityBusinesses } from '../../../models';
+import { getCommunityBusiness } from '../prerequisites';
+import {
+  response,
+  visitActivitiesPostPayload,
+  visitActivitiesPutPayload,
+  id,
+  meOrId,
+  visitActivitiesGetQuery } from './schema';
+import { VisitActivity } from '../../../models/types';
+import { Day } from '../../../types/internal';
+
+interface GetRequest extends Hapi.Request {
+  query: {
+    day: Day
+  };
+}
+
+interface PostOrPutRequest extends Hapi.Request {
+  payload: Partial<VisitActivity>;
+}
+
+export default [
+  {
+    method: 'GET',
+    path: '/community-businesses/{communityBusinessId}/visit_activities',
+    options: {
+      description: 'Retrieve all visit activities for a community business',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: [
+            'visit_activities-own:read',
+            'visit_activities-child:read',
+            'visit_activties-parent:read',
```

Comment:
There's no differentiation between any of these scopes in the handler/pre-reqs. If I only have the `own` scope, I can hit this endpoint for _any_ CB and the handler will still give me the data.